rollup.config.js

```diff
@@ -1,16 +1,16 @@
-import cheap_ts from 'rollup-plugin-cheap-ts';
+import typescript2 from 'rollup-plugin-typescript2';
```

Comment:
I previously preferred `rollup-plugin-typescript2`, but ended up fixing a couple of the most prominent bugs in the official `@rollup/plugin-typescript` and probably prefer it now

Reply:
Sounds good. I'll start switching over to `@rollup/plugin-typescript`.